apps/web/src/sitemap.ts

```diff
@@ -0,0 +1,40 @@
+import type { MetadataRoute } from "next";
+
+const addPathToBaseURL = (path: string) => `https://openstatus.dev${path}`;
+
+export default function sitemap(): MetadataRoute.Sitemap {
+  return [
+    {
+      url: addPathToBaseURL("/"),
+      lastModified: new Date(),
+    },
+    {
+      url: addPathToBaseURL("/play"),
+      lastModified: new Date(),
+    },
+    {
+      url: addPathToBaseURL("/sign-in"),
+      lastModified: new Date(),
+    },
+    {
+      url: addPathToBaseURL("/sign-up"),
+      lastModified: new Date(),
+    },
+    {
+      url: addPathToBaseURL("/sign-in"),
+      lastModified: new Date(),
+    },
+    {
+      url: addPathToBaseURL("/sign-up"),
+      lastModified: new Date(),
+    },
+    {
+      url: addPathToBaseURL("/monitor"),
+      lastModified: new Date(),
+    },
+    {
+      url: addPathToBaseURL("/incident"),
+      lastModified: new Date(),
+    },
```

Comment:
That's our bad. We have moved both paths to `/app/[workspaceId]/incident` and `/app/[workspaceId]/monitor`. Thinking about that, we might not need it for the sitemap as they are used for SEO crawling and those pages are very dynamic and user specific.

The one page that will stay and you can add instead is `/monitor/openstatus`.